RecoHI/HiJetAlgos/src/VoronoiAlgorithm.cc

```diff
@@ -174,6 +174,8 @@ namespace {
 
 	class problem_t {
 	public:
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-const-variable"
```

Comment:
I guess these should have been hidden as visible only for clang?

https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc6_amd64_gcc491/CMSSW_7_4_X_2015-01-17-1400/RecoHI/HiJetAlgos

```
  /build/cmsbuild/jenkins-workarea/workspace/build-any-ib/4713/w/tmp/BUILDROOT/359600e501b3d66139af0f2dcf386d7f/opt/cmssw/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_4_X_2015-01-17-1400/src/RecoHI/HiJetAlgos/src/VoronoiAlgorithm.cc:177:0: warning: ignoring #pragma clang diagnostic [-Wunknown-pragmas]
```